src/libos/src/time/timer_fd.rs

```diff
@@ -0,0 +1,297 @@
+use super::*;
+use crate::fs::{AccessMode, Events, IoctlCmd, Observer, Pollee, Poller, StatusFlags};
+use async_rt::sched::yield_;
+use atomic::{Atomic, Ordering};
+use flume::{Receiver, Sender};
+use std::any::Any;
+use std::sync::mpsc;
+use std::time::Duration;
+use untrusted::{SliceAsMutPtrAndLen, SliceAsPtrAndLen};
+
+#[allow(non_camel_case_types)]
+#[derive(Clone, Copy, Debug, PartialEq)]
+#[repr(u8)]
+enum TimerFdStatus {
+    ONE_SHOT = 0,
+    LOOP = 1,
+    STOP = 2,
+}
+
+#[derive(Debug)]
+pub struct TimerFd {
```

Comment:
To be consistent with other file types (`EpollFile`, `EventFile`, `SocketFile`, etc.), this type should be named `TimerFile`.

Reply:
Sounds reasonal, will do the change.